cmd/operator-sdk/internal/flags.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 The Operator-SDK Authors
```

Comment:
```suggestion
// Copyright 2019 The Operator-SDK Authors
```

And make this its own `internal/flags` package. We'll likely add more global flags here.

Reply:
> And make this its own internal/flags package. We'll likely add more global flags here.

Just a heads up, we already have `pkg/internal/flags` for flags shared between the Ansible and Helm operators.

I'd suggest `internal/pkg/flags` for this, and we can move the others over in a separate PR?

@estroz WDYT?

Reply:
> + // Copyright 2019

Right, I didn't actually look at it, just pulled it from another file. We used to use a "copyright plugin" to keep files updated, is that warranted here?

Reply:
Its probably a good idea going forward, I'll look into it. Do you remember what the plugin is?

Reply:
I created #1367 earlier today if that helps.